README.md

```diff
@@ -1,7 +1,7 @@
 
 # Apache APISIX® Website
 
-[<img align="right" width="150" src="./website/static/img/join-slack.png">](https://apisix.apache.org/docs/general/community)
+[<img align="right" width="150" src="./website/static/img/join-slack.png">](https://apisix.apache.org/docs/general/join)
```

Comment:
Hi, @navendu-pottekkat, it's semantic, but this means we will have many pages 404 🤔 I have some ideas:

1. Keep other URLs, and create a page under `xxx/join.`
2. Do some redirect when visit `/community`, `/subscribe-guide`, and so on.

What do you think?

Reply:
nice catch

Reply:
Seems work fine in preview deployment, which is https://deploy-preview-974--apache-apisix.netlify.app/docs/general/join/

Reply:
After consideration, I could understand why @navendu-pottekkat removes pages like `/community` and `subscribe` and merge them to `/join`, it makes sense to me and is much more straightforward.

To avoid 404 for historical links, I would suggest also supporting redirect `/community` & `/subscribe-xx` to `/join` in the website, cc @SkyeYoung.

Reply:
@juzhiyuan Makes sense. Something like this? https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects

Has this been implemented anywhere currently?

Reply:
I tried adding this plugin but it throws an error "cannot mix different versions of joi schemas". I'm not sure why we are getting this error. Can someone else who has worked on this before offer any suggestions?

Reply:
Don't worry. Could you help list all broken links after this PR gets merged? And we could seek help from @SkyeYoung to implement this.

Reply:
Yep. I could do that. Let me try changing the versions and see if I'm able to fix this. If not, I will work with @SkyeYoung and setup the redirects.